common/cards/alter-egos/single-use/egg.ts

```diff
@@ -49,7 +51,7 @@ class EggSingleUseCard extends SingleUseCard {
 					if (!pickResult.card) return 'FAILURE_INVALID_SLOT'
 
 					// Store the row index to use later
-					player.custom[targetKey] = rowIndex
+					player.custom[targetKey] = pickResult
```

Comment:
What does this change? Looks like it'd probably fix a targeting bug, if yes, what is it as we'd want to add it in the next changelog

Reply:
I'm not sure. This just made egg easier to program with the new log changes. Egg was written a bit strangely before. Would appreciate if you could send me a link to the bug report so I can confirm it would fix it.

Reply:
I'm not aware of a bug report, but this needs testing to clarify what it might change

Reply:
It doesn't change anything that I'm aware of (besides for the bug that screaper found).